packages/web/src/lib/myalgowallet-mode.ts

```diff
@@ -89,6 +89,38 @@ export class MyAlgoWalletSession {
 		}
 	}
 
+	/**
+	 * @async
+	 * @description Signs a teal program (https://algorand.github.io/js-algorand-sdk/modules.html#tealSign)
+	 * @param logic Teal program
+	 * @param address Signer Address
+	 * @returns Returns signed teal
+	 * for more info: https://connect.myalgo.com/docs/interactive-examples/TealSign
+	 */
+	async signLogicSignature(logic: string | Uint8Array, address: string): Promise<Uint8Array> {
```

Comment:
we are not creating the txn here, rather we are signing it, so shouldn't be sign.?

Reply:
we are creating a logic signature, not a transaction. Also, let's use `Sig`, rather than `Signature`. Few other options:
* mkLogicSig
* createLogicSig
* createSmartSig
* signLogic

Reply:
okay will update it to `signLogic`